index.html

```diff
@@ -1,198 +1,309 @@
-            <div><img src="assets/separator.png"  alt=""/></div>
+          </div>
+          <ul class="btn">
+            <li><a href="#">Get Resume</a></li>
+          </ul>
+          <p>&nbsp;</p>
+          <div class="specials">
+            <img src="assets/Languages.png" alt="" /><a href="#"
+              ><i class="fa fa-angle-down icons" aria-hidden="true"></i
+            ></a>
+          </div>
+          <br />
+          <div class="langs">
+            <ul>
+              <li>
+                <a href="#"
+                  ><img src="assets/js-logo.png" alt="" /> &nbsp; &nbsp;
+                  JavaScript</a
+                >
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assets/html-logo.png" alt="" /> &nbsp;
+                  &nbsp;HTML</a
+                >
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assets/css-logo.png" alt="" /> &nbsp; &nbsp; CSS</a
+                >
+              </li>
+            </ul>
+          </div>
+          <br />
+          <div class="specials">
+            <img src="assets/Frameworks.png" alt="" /><a href="#"
+              ><i class="fas fa-greater-than icons"></i
+            ></a>
+          </div>
+          <div><img src="assets/separator.png" alt="" /></div>
+          <div class="specials">
+            <img src="assets/Skills.png" alt="" /><a href="#"
+              ><i class="fas fa-greater-than icons"></i
+            ></a>
+          </div>
+          <div><img src="assets/separator.png" alt="" /></div>
+        </div>
+      </section>
+
+      <!-- Contact Form Section -->
+      <section>
+        <div class="wrapper">
+          <div class="background-image"><img src="card-cover.png" alt="" />
+            <h1>Contact me</h1>
+            <p>If you have an application you
+              are interested in developing, a feature that you need built or a
+              project that needs coding. I’d love to help with it
+            </p>
+
+            <form
+              action="https://formspree.io/f/xoqrrrbp"
+              method="POST">
+             <div class="form-controls">
+               <input
+                 type="text"
+                 name="text-chars"
+                 maxlength="30"
+                 placeholder="Esi Ogagaoghene"
+                 required
+               />
+               <input
+                 type="email"
+                 name="email-chars"
+                 placeholder="esi.ogagaoghene@yahoo.com"
+                 required
+               />
+               <textarea
+                   rows="10"
+                   cols="20"
+                   name="textarea-chars"
+                   maxlength="500"
+                   placeholder="Write your message"
+                   required></textarea>
+               <div>
+                 <button type="submit" class="form-button">Get In Touch</button>
```

Comment:
- Please implement the Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.